Nos/Views/Profile/ProfileHeader.swift

```diff
@@ -128,13 +155,49 @@ struct ProfileHeader: View {
                             followers: followers
                         )
                     }
+                    .padding(.top, 5)
+                }
+
+                HStack(spacing: 34) {
```

Comment:
I think the design is divided into two halves: one half for the Follow or Edit Profile button, and the other half for the following and relays. If that's true, the spacing of 34 wouldn't make sense, but I don't know for sure that I'm right. Could we ask Sebastian about this?